tests/test_web_log.py

```diff
@@ -43,13 +43,31 @@ def test_access_logger_format() -> None:
     Ref: https://stackoverflow.com/a/46102240/595220
     """,  # noqa: E501
 )
-def test_access_logger_atoms(mocker) -> None:
-    utcnow = datetime.datetime(1843, 1, 1, 0, 30)
+def test_access_logger_time(mocker) -> None:
```

Comment:
plz use `monkeypatch` instead
```suggestion
def test_access_logger_time(monkeypatch) -> None:
```
`pytest-mock` has certain probems which I don't want us to hit.

Reply:
monkeypatch can't set attributes of built-in/extension type 'datetime.datetime'.
is there a nice way?